From a4f0a0d0a88ccd2a713ca3197ab23dc4ef059d69 Mon Sep 17 00:00:00 2001
From: Damien Caliste <dcaliste@free.fr>
Date: Tue, 11 Dec 2018 13:21:35 +0100
Subject: [PATCH 06/11] Disable various dependencies in plugins.

---
 plugins/textshape/TextTool.cpp | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/plugins/textshape/TextTool.cpp b/plugins/textshape/TextTool.cpp
index c5ab189ec42..d1852cb41dd 100644
--- a/plugins/textshape/TextTool.cpp
+++ b/plugins/textshape/TextTool.cpp
@@ -76,8 +76,10 @@
 //#include <ResizeTableCommand.h>
 #include <KoIcon.h>
 
+#ifdef HAVE_KRUN
 #include <krun.h>
 #include <kstandardshortcut.h>
+#endif
 #include <kactionmenu.h>
 #include <kstandardaction.h>
 #include <ksharedconfig.h>
@@ -2914,6 +2916,7 @@ void TextTool::runUrl(KoPointerEvent *event, QString &url)
         QMimeDatabase db;
         QString type = db.mimeTypeForUrl(_url).name();
 
+#ifdef HAVE_KRUN
         if (KRun::isExecutableFile(_url, type)) {
             QString question = i18n("This link points to the program or script '%1'.\n"
                                     "Malicious programs can harm your computer. "
@@ -2924,10 +2927,13 @@ void TextTool::runUrl(KoPointerEvent *event, QString &url)
             if (choice != KMessageBox::Yes)
                 return;
         }
+#endif
     }
 
     event->accept();
+#ifdef HAVE_KRUN
     new KRun(_url, 0);
+#endif
 }
 
 void TextTool::debugTextDocument()
-- 
2.20.1

