From 4bb4d98c05e08f90ef9b1725bef60c64137f3206 Mon Sep 17 00:00:00 2001
From: Damien Caliste <dcaliste@free.fr>
Date: Tue, 11 Dec 2018 13:27:28 +0100
Subject: [PATCH 12/14] Reenable transparent background.

---
 words/part/KWCanvasItem.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/words/part/KWCanvasItem.cpp b/words/part/KWCanvasItem.cpp
index 1edd07e1764..247518b0f44 100644
--- a/words/part/KWCanvasItem.cpp
+++ b/words/part/KWCanvasItem.cpp
@@ -162,7 +162,7 @@ void KWCanvasItem::updateCanvas(const QRectF &rc)
 
 void KWCanvasItem::paint(QPainter *painter, const QStyleOptionGraphicsItem *option, QWidget *)
 {
-    painter->fillRect(option->exposedRect, QColor::fromRgb(232, 233, 234));
+    // painter->fillRect(option->exposedRect, QColor::fromRgb(232, 233, 234));
     KWCanvasBase::paint(*painter, option->exposedRect);
 }
 
-- 
2.23.0

