From 93f5ab5db6210f6583996236ed6b79e912064773 Mon Sep 17 00:00:00 2001
From: Damien Caliste <dcaliste@free.fr>
Date: Tue, 11 Dec 2018 13:26:34 +0100
Subject: [PATCH 08/10] Enable page caching to improve performences.

---
 words/part/KWCanvasBase.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/words/part/KWCanvasBase.cpp b/words/part/KWCanvasBase.cpp
index f20322ab14b..7e9a35ee602 100644
--- a/words/part/KWCanvasBase.cpp
+++ b/words/part/KWCanvasBase.cpp
@@ -70,7 +70,7 @@ KWCanvasBase::KWCanvasBase(KWDocument *document, QObject *parent)
 {
     m_shapeManager = new KoShapeManager(this);
     m_toolProxy = new KoToolProxy(this, parent);
-    //setCacheEnabled(true);
+    setCacheEnabled(true);
 }
 
 KWCanvasBase::~KWCanvasBase()
-- 
2.20.1

